### PR TITLE
update `rpc_mode` default value

### PR DESCRIPTION
**Pull request by smk762**
closes https://github.com/KomodoPlatform/komodo-docs-mdx/issues/265

### CODE REVIEW

**Comment by smk762** [comment]
@shamardy which activation methods accept this param? I will apply updates to postman collection.

**Comment by shamardy** [comment]
> which activation methods accept this param? I will apply updates to postman collection.

`enable_eth_with_tokens` method only

File: src/pages/komodo-defi-framework/api/v20/enable_eth_with_tokens/index.mdx [hunk 1]
```diff
@@ -67,7 +67,6 @@ Additionally, it supports ERC20 tokens on the ETH chain and associated ERC20 lik
           "gui_auth": true
         }
       ],
-      "rpc_mode": "Http",
```
Comment (AndrewDelaney) [inline src/pages/komodo-defi-framework/api/v20/enable_eth_with_tokens/index.mdx:70]: This section is the "Request including optional gas station parameters" section. Should we not still display the optional parameter with its value set to 'default' instead?

Comment (smk762) [reply src/pages/komodo-defi-framework/api/v20/enable_eth_with_tokens/index.mdx:70]: As it defaults to `Default`, seems a waste to define what's defined by default. Might be better to add an example with the non-default value.

**Review by ShantanuSharma9873 — APPROVED** [review]

**Review by shamardy — APPROVED** [review]

**Review by gcharang — APPROVED** [review]

**Pull request merged**